From 4d10b52399b9a24a750be76f377c7f4426c69972 Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Wed, 20 Nov 2019 10:05:07 -0600
Subject: [PATCH 08/34] ARM64: dts: lpd-imx8mm: Remove unused functions

The baseboard has provisions for both HDMI and LVDS bridges, but
only one can be enabled at once, so the bridge chips should not
be included in a common file.

This patch removes the references to the LCD controller, DSI, and
the bridge chips from the baseboard common file.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 .../dts/freescale/lpd-imx8mm-baseboard.dtsi   | 34 -------------------
 1 file changed, 34 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi b/arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi
index f48f6ef7f700..7c68f653b92c 100644
--- a/arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi
+++ b/arch/arm64/boot/dts/freescale/lpd-imx8mm-baseboard.dtsi
@@ -108,12 +108,6 @@
 			>;
 		};
 
-		pinctrl_mipi_dsi_en: mipi_dsi_en {
-			fsl,pins = <
-				MX8MM_IOMUXC_GPIO1_IO08_GPIO1_IO8		0x16
-			>;
-		};
-
 		pinctrl_i2c2_synaptics_dsx_io: synaptics_dsx_iogrp {
 			fsl,pins = <
 				MX8MM_IOMUXC_GPIO1_IO09_GPIO1_IO9		0x19	/* Touch int */
@@ -251,20 +245,6 @@
 	pinctrl-0 = <&pinctrl_i2c2>;
 	status = "okay";
 
-	adv_bridge: adv7535@3d {
-		compatible = "adi,adv7533";
-		reg = <0x3d>;
-		adi,addr-cec = <0x3b>;
-		adi,dsi-lanes = <4>;
-		status = "okay";
-
-		port {
-			adv7535_from_dsim: endpoint {
-				remote-endpoint = <&dsim_to_adv7535>;
-			};
-		};
-	};
-
 	ov5640_mipi: ov5640_mipi@10 {
 		compatible = "ovti,ov5640_mipi";
 		reg = <0x10>;
@@ -337,20 +317,6 @@
 		};
 };
 
-&lcdif {
-	status = "okay";
-};
-
-&mipi_dsi {
-	status = "okay";
-
-	port@1 {
-		dsim_to_adv7535: endpoint {
-			remote-endpoint = <&adv7535_from_dsim>;
-		};
-	};
-};
-
 &mu {
 	status = "okay";
 };
-- 
2.17.1

